ui/src/__tests__/components/microsegmentation/__snapshots__/AddSegmentation.test.js.snap

```diff
@@ -1101,7 +1101,7 @@ exports[`AddSegmentation should render fail to submit add-segmentation: destinat
           <label
             for="checkbox-validate-policytrue"
           >
-            Validate Microsegmentation policy against PES network policy
+            Validate Microsegmentation policy against PES network policy (OnPrem)
```

Comment:
Maybe replace (OnPrem) to (relevant only for policies with On Prem hosts only) or something like that?